language-server/modules/langserver-core/src/test/resources/completion/statement_context/source/return_stmt_ctx_source12.bal

```diff
@@ -0,0 +1,9 @@
+import ballerina/module1;
+
+public listener module1:Listener l1 = new(9090);
+
+service testService on l1 {
+    resource function testFunction . () {
```

Comment:
A resource function's syntax is
```
resource function <http_request_method> <path> () {
}
```
Therefore, need to change `testFunction` a vaid http method like `get`, `post`.